src/odemis/driver/hamamatsurx.py

```diff
@@ -625,15 +640,122 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
         timeRange = util.find_closest(timeRange, choices)  # make sure value is in choices
         self.timeRange = model.FloatEnumerated(timeRange, choices, setter=self._setTimeRange, unit="s")
 
+        pllMode = self.GetPLLMode()
+        if pllMode is not None:
+            self.phaseLocked = model.BooleanVA(pllMode, setter=self._setPLLMode)
+
         # a variable that stores the current timeRange conversion for e.g. the scaling table conversion
         # is set in the setter of the timeRange VA
         self.timeRangeFactor = None
 
         # read-only VAs TODO: Trig. Mode, Trig. level, Trig. slope?
 
+    def _updateSettings(self) -> None:
+        """
+        Read all the current streak unit settings from the RemoteEx and reflect them on the VAs
+        """
+        logging.debug("Updating streak unit settings")
+        try:
+            timeRange = self.GetTimeRange()
+            if timeRange != self.timeRange.value:
+                self.timeRange._value = timeRange
+                self.timeRange.notify(timeRange)
+
+            gain = self.GetMCPGain()
+            if gain != self.MCPGain.value:
+                self.MCPGain._value = gain
+                self.MCPGain.notify(gain)
+                logging.debug("MCP Gain synced with RemoteEx.")
```

Comment:
Let's drop this log, as the other settings don't have a log entry, and I don't think it's really important per se to know.